### ISSUE TITLE
Some problems about the MAP of train and val datasets

### ISSUE THREAD

**Issue by SYSU-LamTou** [zylo117/Yet-Another-EfficientDet-Pytorch#296]
There are about 15,000 images and 44 kinds.
I train the D7 model with pretrained weights. After about 30 epochs, the loss is stable. The train loss is about 0.02 and the val loss is 0.4. 
But the Map of train set is quite low, about 68%, and the val set is 48%.
I keep on training for more than 10 epoch, both the train and val Map don't increase or decrease. I also try the SGD, but it seems doesn't work. The learning rate is 1e-5.
What's the problems and how can I improve the model? At least the Map of train set should be over 90%..?
Thanks!!!

**Comment by SYSU-LamTou** [comment]
oh~~ 
Also, why the train's loss is quite low, but the Map is just 68%?

**Comment by zylo117** [comment]
1. it's obiviously overfitting.
2. loss will be faking low if anchors don't match. Some gt boxes will be not matched so their losses will not be considered.

**Comment by SYSU-LamTou** [comment]
> it's obiviously overfitting.
> loss will be faking low if anchors don't match. Some gt boxes will be not matched so their losses will not be co

**Comment by SYSU-LamTou** [comment]
> it's obiviously overfitting.
> loss will be faking low if anchors don't match. Some gt boxes will be not matched so their losses will not be considered.

thanks a lot!
